packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -52,6 +52,8 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
   private _root: HTMLElement;
   private _id: string;
   private _activeElement: HTMLElement | null;
+  /** The active element if there is one, otherwise the first focusable element. */
```

Comment:
I've tried to clarify the docs here. The basic idea is to only set `_activeElement` when a child of the FocusZone is actually focused, and not default it to the first focusable element. `_defaultElement` will be `_activeElement` if it is not null or the first focusable element otherwise.